chelsea-shayne/styles/modules.css

```diff
@@ -0,0 +1,72 @@
+.site-header {
+  background: #FFEAAA;
+}
+
+/* REVIEW: Float the main header to the left, to make room for the hamburger menu. */
+.heading-group-h1 {
+  float: left;
+  padding: 0 0 .2em 1em;
+}
+
+.icon-menu {
+  float: right;
+  font-size: 3em;
+}
+
+.main-nav {
+  padding: 0.1em;
+}
+
+.main-nav ul {
+  display: none;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+.main-nav:hover ul {
+  display: block;
+}
+
+.main-nav li {
+  margin-right: .4em;
+  font-size: 1.5em;
+  padding: .5em .2em;
+  border-bottom: 1px solid #ccc;
+}
+
+#filters {
+  padding: 0;
+  list-style: none;
+}
+
+.template {
+  display: none;
+}
+
+.byline {
+  color: #777;
+}
+
+.byline a {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+@media only screen and (min-width: 480px){
+  .icon-menu:before {
+    display: none;
+  }
+
+  .main-nav ul {
+    clear: both;
+    display: inline;
+    margin: 0;
+  }
+
+  .main-nav li {
```

Comment:
The hover hasn't been cleared in the media queries, so when the screen gets larger than 480px and you hover on the UL it moves around on the page.